mlflow/keras.py

```diff
@@ -311,3 +316,64 @@ def load_model(model_uri, **kwargs):
         local_model_path,
         flavor_conf.get("data", _MODEL_SAVE_PATH))
     return _load_model(model_path=keras_model_artifacts_path, keras_module=keras_module, **kwargs)
+
+
+class __MLflowKerasCallback(Callback):
+    """
+    Callback for auto-logging metrics and parameters.
+    Records available logs after each epoch.
+    Records model structural information as params after training finishes.
+    """
+
+    def __enter__(self):
```

Comment:
Do we actually need the context manager methods (`__enter__` and `__exit__`)? Doesn't look like they do anything

Reply:
True, removed